Makefile

```diff
@@ -133,7 +133,7 @@ SHELL = /usr/bin/env bash -o pipefail
 
 LABEL_FILTERS ?=
 JUNIT_REPORT_FILE ?= "junit.e2e_suite.1.xml"
-GINKGO_SKIP ?=
+GINKGO_SKIP ?= "clusterctl-Upgrade"
```

Comment:
instead of this, another way is more ginkgo v2 way. where we can specificy an appropriate label filter https://onsi.github.io/ginkgo/#skipping-specs eg <pre>

ginkgo --label-filter="!slow" | Avoid any specs labelled slow
</pre>